Software role permissions updates (#1403)
astronomer/docs: software_versioned_docs/version-0.29/manage-platform-users.md

---
title: 'Manage users and roles on Astronomer Software'
sidebar_label: 'Manage users and roles'
id: manage-platform-users
description: Add and customize user permissions on Astronomer Software.
---

This is where you'll find information about role management and guidelines around public signups, role customization, and adding System Admins.

## Add users to Astronomer

> When Astronomer Software is first deployed, the first user to log in is granted System Admin permissions by default. From there, a user can created on Astronomer Software through one of the following methods:

Reviewer: ```suggestion
When Astronomer Software is first deployed, the first user to log in is granted System Admin permissions by default. From there, a user can be created on Astronomer Software through one of the following methods:
```

Then, push the configuration change to your platform as described in [Apply a config change](apply-platform-config.md).

## User roles on Astronomer

System Admins can customize permissions across teams. On Astronomer, users can be assigned roles at 2 levels:

- Workspace Level (Viewer, Editor, Admin)
- System Level (Viewer, Editor, Admin)

> Workspace roles apply to all Deployments within a single Workspace, while system roles apply to all Workspaces across a single cluster. For a detailed breakdown of the 3 Workspace-level roles on Astronomer (Viewer, Editor and Admin), read [Manage user permissions on an Astronomer Workspace](workspace-permissions.md).

Reviewer: ```suggestion
Workspace roles apply to all Deployments within a single Workspace, while system roles apply to all Workspaces across a single cluster. For a detailed breakdown of the three Workspace-level roles on Astronomer Software (Viewer, Editor and Admin), see [Manage user permissions on an Astronomer Workspace](workspace-permissions.md).
```

<!--- Version-specific -->

> Before customizing role permissions, view [default Houston API configuration](https://github.com/astronomer/docs/tree/main/software_configs/0.29/default.yaml) and answer the following questions:

Reviewer: ```suggestion
Before customizing role permissions, review [default Houston API configuration](https://github.com/astronomer/docs/tree/main/software_configs/0.29/default.yaml) and then answer the following questions:
```

> - What role do you want to configure? (e.g. `DEPLOYMENT_EDITOR`)

Reviewer: ```suggestion
- What role do you want to configure? For example, `DEPLOYMENT_EDITOR`.
```

> - What permission(s) would you like to add to or remove from that role? (e.g. `deployment.images.push`)

Reviewer: ```suggestion
- What permission(s) do you want to add to or remove from the role? For example, `deployment.images.push`.
```

## System roles

> The System Admin role on Astronomer Software brings a range of cluster-wide permissions that supersedes Workspace-level access and allows a user to monitor and take action across Workspaces, Deployments and Users within a single cluster.

Reviewer: ```suggestion
The System Admin role on Astronomer Software brings a range of cluster-wide permissions that supersedes Workspace-level access and allows a user to monitor and take action across Workspaces, Deployments, and Users within a single cluster.
```
